### ISSUE TITLE
Upgraded to 'null'

### ISSUE THREAD

**Issue by eneroth** [liquidz/antq#42]
Antq will sometimes update to "null" for deps like ions/dev-local or git deps. I imagine "null" will rarely be a real version, so perhaps it would be safe to skip these updates?

```
Upgraded com.datomic/ion '0.9.48' to 'null' in deps.edn.
Upgraded expound/expound 'bdccedc96c65e0d104e6139121a501453007f398' to 'null' in deps.edn.
Upgraded com.datomic/dev-local '0.9.225' to 'null' in deps.edn.
Upgraded com.wsscode/pathom-datomic '88212bf72b3092aa12af1c7aa94eb696fc08d12c' to 'null' in deps.edn.
Upgraded com.cognitect/rebl '0.9.242' to 'null' in deps.edn.
Upgraded com.google.guava/guava '22.0' to '30.0-jre' in deps.edn.
```

**Comment by liquidz** [comment]
@eneroth Thanks for your reporting!

Oh sorry, it's a known bug (I forgot to open a issue)

> it would be safe to skip this updates?
Right. I'll fix to skip upgrading artifacts which cannot fetch the version.

**Comment by liquidz** [comment]
@eneroth I fixed in `dev` branch.
Could you try following steps?

- run `make uberjar` to generate target/antq-standalone.jar
- run `java -jar /path/to/your/antq/repo/target/antq-standalone.jar` in your project directory

**Comment by eneroth** [comment]
Checked out 8876d3ad9f02fc2fbe5207e26c2312282523c0b9, did the above with `--upgrade` and `--force`, and it bumped the one dep it could fetch a new version for, skipping the three duds. 👍

It seems to work as intended.

**Comment by liquidz** [comment]
Thanks for your confirmation!

**Comment by liquidz** [comment]
Just released 0.9.2 #43
